.asf.yaml

```diff
@@ -1,5 +1,20 @@
-# https://cwiki.apache.org/confluence/display/INFRA/.asf.yaml+features+for+git+repositories
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
@pjfanning I think you can remove this header change because its going to be done properly in https://github.com/apache/incubator-pekko/pull/50